assets/scss/components/_button.scss

```diff
@@ -257,6 +258,17 @@ Class                   | Description
 	background: $C_contentBG;
 }
 
+.button--reset {
+	@include buttonHover(transparent);
+	background: transparent;
```

Comment:
`.button[disabled]` has a lot of `!important` going on, including `background`, so this one will get beaten on disabled buttons